### PR TITLE
Added Tutorials for Google Batch

### PR DESCRIPTION
**Pull request by zyosufzai**
### Description

*Added tutorials for using Google Batch based on the LS API Tutorials, added usage of Google Batch in ReadMe, added warnings in the LS API tutorials that the product is depreciated and linked to the Google Batch tutorials, and added warning in Snakemake tutorials that LS API is depreciated and to check Snakemake documentation for when they have instuctions on Google Batch (right now they only use LS API same goes for Cromwell)*

### Assignee

*Assignees: @kyleoconnell-NIH *

## PR checklist
*Please ensure the following:*
- [X] This comment contains a description of changes (with reason).
- [X] All changes were tested.
- [X] If you've fixed a bug mention the issue number/name.
- [X] Apply approriate tags (e.g. documentation, bug)

### CODE REVIEW

File: tutorials/notebooks/GoogleBatch/nextflow/batch_images/service_account_4.jpg
Comment (kyleoconnell-NIH) [inline tutorials/notebooks/GoogleBatch/nextflow/batch_images/service_account_4.jpg:1]: Can you black out the Principal and the project name?

File: tutorials/notebooks/GoogleBatch/nextflow/batch_images/service_account_1.jpeg
Comment (kyleoconnell-NIH) [inline tutorials/notebooks/GoogleBatch/nextflow/batch_images/service_account_1.jpeg:1]: Block out the project name

File: tutorials/notebooks/GoogleBatch/nextflow/batch_images/Job_l.png
Comment (kyleoconnell-NIH) [inline tutorials/notebooks/GoogleBatch/nextflow/batch_images/Job_l.png:1]: Block project name

File: tutorials/notebooks/LifeSciencesAPI/snakemake/LS_API_Snakemake.ipynb [hunk 1]
```diff
@@ -523,9 +525,9 @@
    "uri": "gcr.io/deeplearning-platform-release/base-cpu:m93"
   },
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "PySpark (Local)",
```
Comment (kyleoconnell-NIH) [inline tutorials/notebooks/LifeSciencesAPI/snakemake/LS_API_Snakemake.ipynb:528]: Why are we using PySpark Kernel?

**Review by kyleoconnell-NIH — CHANGES_REQUESTED** [review]
Did you also convert the Nextflow Tutorial under LS API? to Batch?

**Comment by zyosufzai** [comment]
> Did you also convert the Nextflow Tutorial under LS API? to Batch?

yes I added a Batch tutorials for Nextflow and added a warning in the LS API that it is depreciated and linked back to the Batch tutorials. I didn't remove the LS API tutorials because technically you can still use LS API until 2025

**Review by kyleoconnell-NIH — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -65,6 +66,9 @@ To spin up a Notebook instance and import an example training notebook, follow o
 
 If you want to practice using the terminal or review BASH commands in Jupyter, look at [this module](https://github.com/NIGMS/IntroBioinformaticsDartmouth) from Dartmouth developed for the NIGMS Sandbox. 
 
+## **Generative AI in Vertex AI** <a name="genai"></a>
+Vertex AI not only has the capabilities to create Jupyter notebooks but also generative AI models. Some examples of what generative AI models are capabile of are extracting wanted information from text, transforming speech into text, generating images from describtions and vice versa, and so much more. Vertex AI's [Generative AI Studio](https://cloud.google.com/vertex-ai/docs/generative-ai/learn/generative-ai-studio) console allows the user to rapidly create, test, and train generative AI models on the cloud in a safe and secure setting. The studio also has ready to use models all contained with in the [Model Garden](https://cloud.google.com/vertex-ai/docs/start/explore-models), these models range from foundation models, fine-tunable models, and task-specific solutions.
```
[the remark this answers is not in the document]
Comment (zyosufzai) [reply README.md]: resolved with the latest push

File: tutorials/notebooks/GenAI/GenAIStudioGCP.ipynb
Comment (petersonjdNIH) [inline tutorials/notebooks/GenAI/GenAIStudioGCP.ipynb:1]: @zyosufzai Is this deep link to the Google Cloud Console safe for the public, or is it deep linking to something that only belongs to NIH?

Comment (zyosufzai) [reply tutorials/notebooks/GenAI/GenAIStudioGCP.ipynb:1]: Do you mean this link 'https://console.cloud.google.com/vertex-ai/generative/language?_ga=2.182664366.923116401.1692009977-1042353744.1691708677'? No this is not NIH specific it is a link to the Generative AI Studio. If you'd like I can instead replace this with instruction on how to get to this page instead (e.g. Generative AI Studio > Language > Article Summary)

**Review by petersonjdNIH — APPROVED** [review]

**Pull request merged**